src/Common/Commands.Common/PowerShellUtilities.cs

```diff
@@ -93,5 +93,16 @@ public static IEnumerable<RuntimeDefinedParameter> GetUsedDynamicParameters(Runt
         {
             return dynamicParameters.Values.Where(dp => MyInvocation.BoundParameters.Keys.Any(bp => bp.Equals(dp.Name)));
         }
+
+        public static string FormatErrorRecord(ErrorRecord record)
```

Comment:
@jaredmoo any reason a simple format wrapper function has to be introduced in the product.
if you need it for test reason, declare it locally inside EnvironmentSetupHelper.